.github/ISSUE_TEMPLATE/release-notes-subtask-template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Release Notes Subtask
+about: Template to create a subtask for the release notes in each iteration.
+title: 'docs: 📚 create release notes for iteration #[NUMBER]'
+labels: 'documentation, 🎨 figma, 🔧 code, Subtask'
+assignees: ''
+
+---
+
+## Description
+In order to publish our release notes by the end of every iteration in MS Teams and in UniVersum, they need to be created on our communications page.
+
+## ToDos
+- [ ] create the release notes in figma
+- [ ] create an publish the page in our communications repo
+  - [ ] create iteration page
+  - [ ] update select filed with new option (latest iteration)
+  - [ ] update all solid resources in the repo to the latest version
```

Comment:
Can you clarify this part @karlbaumhauer? Which resources do we need to update?

Reply:
updated it... you think it´s enough?